packages/terra-props-table/src/PropsTable.jsx

```diff
@@ -75,7 +75,7 @@ const PropsTable = ({ componentName, src, ...customProps }) => {
   return (
     <div dir="ltr" className="markdown-body">
       <h2>{componentName} Props</h2>
-      <table {...customProps}>
+      <table className={cx('props-table')} {...customProps}>
```

Comment:
Classnames and customProps should be flipped, if custom props includes a classname it will always overwrite any className declared before it when it spreads.

```
<table {...customProps} className={cx('props-table')}>
```

We should probably also be merging the customProps classNames into our classNames. Here is an example: https://github.com/cerner/terra-core/blob/ada5669999f1f56efd8420e3e296716a681d7643/packages/terra-card/src/Card.jsx#L21-L26